0/setup.md

```diff
@@ -16,6 +16,9 @@ curl https://getsubstrate.io -sSf | bash -s -- --fast
 ```
 rustup target add wasm32-unknown-unknown --toolchain stable
 rustup component add rust-src --toolchain nightly
+rustup toolchain install nightly-2020-06-01
+rustup target add wasm32-unknown-unknown --toolchain nightly-2020-06-01
+cargo +nightly-2020-06-01 build --release
```

Comment:
Did you mean to add a build command here? This seems like still the setup stage and maybe there is nothing to be built yet.

Reply:
Yeah, this is not correct.

Reply:
No I literally just copied-and-pasted some commands and figured we could hash it out in code review :sweat_smile: Good looking out :+1: